client/client/main.go

```diff
@@ -140,12 +162,15 @@ func main() {
 	reader := bufio.NewReader(os.Stdin)
 
 	for {
-		fmt.Print("Please enter word to search for (enter to exit): ")
-		keyword, _ := reader.ReadString('\n')
-		keyword = keyword[:len(keyword)-1]
-		if keyword == "" {
+		fmt.Print("Please enter words to search for separated by spaces (enter to exit): ")
+		input, _ := reader.ReadString('\n')
+		input = strings.TrimRight(input, "\n")
+		if input == "" {
 			break
 		}
-		performSearchWord(cli, keyword)
+		keywords := strings.Split(input, " ")
+		for _, keyword := range keywords {
+			performSearchWord(cli, keyword)
```

Comment:
Ah, sorry I missed there's a header.  Still doesn't seem too useful to me, but ok.